lib/README.md

```diff
@@ -92,20 +114,40 @@ JWT=$(curl -s -X POST http://localhost:8000/auth \
 -d '{"username": "admin", "passwd" : "admin"}' | jq -r ".jwt")
 ```
 
-Note: the previous command requires the `jq` command, you can install it on ubuntu
-with the following command:
+Note: the previous command requires the `jq`and `curl` command, you can install it on Debian-based Linux distributions with the following command:
 ```
 sudo apt-get install jq
+sudo apt-get install curl
 ```
 
-Add the node to Dojot.
+How to add a node to Dojot:
+
 ```sh
 curl -H "Authorization: Bearer ${JWT}" http://localhost:8000/flows/v1/node -H 'content-type: application/json' -d '{"image": "<your dockerHub username>/<node name>:<image tag>", "id":"<node name>"}'
 ```
-
 Now the node will be available on the FlowBroker Dojot's interface.
 
+#### Making changes to the source code of the node and seeing them reflected
+
+**Note:** If you need to the source code of the node and seeing them reflected, you need to: **rebuild the container**; **push it to the registry again with a new tag**; **remove the node from dojot via api**; **and add the node to dojot again**. Always change the `<image tag>` of the docker image, to force the update and see your changes reflected.
+
+How to remove a node from Dojot:
+
+```sh
+curl -X DELETE -H "Authorization: Bearer ${JWT}" http://localhost:8000//flows/v1/node/<node name> -H 'content-type: application/json'
+```
+
+**ATTENTION**: The `id` to add the node via API (when request `/flows/v1/node`) must be the same as `name` and `id` defined in `getMetadata` in the class that extends `dojot.DataHandlerBase`. And within the html called in the `getNodeRepresentationPath` method also in the class that extends `dojot.DataHandlerBase`, the references inside the html `data-template-name=`, `data-help-name=`, `registerType(..` ,  and `RED._("...` must have this same `id`/`name`.
```

Comment:
This paragraph is kind of confusing. My suggestion is to create a list of places where the user should make sure that the id/name matches.